zerver/lib/actions.py

```diff
@@ -333,6 +368,7 @@ def do_set_realm_name(realm, name):
     return {}
 
 def do_set_realm_restricted_to_domain(realm, restricted):
+    # type: (Realm, bool) -> Dict[None, None]
```

Comment:
I can change `json_response` in zerver/lib/response.py to accept `None`. This is my fix:
```diff
 def json_response(res_type="success", msg="", data={}, status=200):
     content = {"result": res_type, "msg": msg}
-    content.update(data)
+    if data:
+        content.update(data)
     return HttpResponse(content=ujson.dumps(content) + "\n",
                         content_type='application/json', status=status)
```